.babelrc

```diff
@@ -1,9 +1,11 @@
 {
   "presets": ["react", "es2015"],
-  "plugins": ["add-module-exports"],
   "env": {
     "development": {
       "presets": ["react-hmre"]
+    },
+    "production": {
+      "plugins": ["transform-es2015-modules-umd"]
```

Comment:
I moved this under the production environment because the `react-hmre`modules were ending up in the UMD block of the built file.